src/Illuminate/Http/Resources/ConditionallyLoadsAttributes.php

```diff
@@ -188,7 +188,7 @@ protected function whenPivotLoaded($table, $value, $default = null)
     protected function transform($value, callable $callback, $default = null)
     {
         return transform(
-            $value, $callback, func_num_args() === 3 ? $default : new MissingValue
+            $value, $callback, func_num_args() === 3 ? value($default) : new MissingValue
```

Comment:
The `transform()` global helper function already calls the `Closure` in that `$default` argument:

```php
        if (is_callable($default)) {
            return $default($value);
        }
```

```
>>> transform('', function () { return 'foo'; }, function () { return 'bar'; })
=> "bar"
```